docs/readme.wwwallet_setup.md

```diff
@@ -0,0 +1,138 @@
+## Prerequisites
+- Docker and Docker Compose installed on your machine.
+- An instance of Mysql greater or equal to version 8.
+
+## Setup Instructions
+To set up an instance of wwwallet, the source code of the frontend and backend must be cloned from the respective GitHub repositories.
+The frontend and backend can be run using Docker. Below are the steps to set up and run wwwallet.
+
+1. Clone the repositories:
+```bash
+git clone https://github.com/wwWallet/wallet-backend-server.git
```

Comment:
please integrate wallet backend server and frontend within the container of the docker compose